File: Networking/Response/AllEventsResponse.cs
using mpp_proiect_csharp_DianaGliga11.Model;

namespace Networking.Response
{
    public class AllEventsResponse : IResponse
    {
        public List<Event> events{ get; set; }

        public AllEventsResponse()
        {
        }

        public AllEventsResponse(List<Event> events)
        {
            this.events = events;
        }
    }
}

File: Networking/src/main/java/response/AllEventsResponse.java
package response;

import example.model.Event;

import java.util.Collection;

public class AllEventsResponse implements Response {
    private final Collection<Event> events;

    public AllEventsResponse(Collection<Event> events) {
        this.events = events;
    }

    public Collection<Event> getEvents() {
        return events;
    }
}
